akka-actor/src/main/scala/akka/util/ConstantFun.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.util
+
+import akka.annotation.InternalApi
+import akka.japi.function.{ Function ⇒ JFun, Function2 ⇒ JFun2 }
+import akka.japi.{ Pair ⇒ JPair }
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] object ConstantFun {
```

Comment:
Hm, good point - OR Akka HTTP should not touch it and use it's own place for those would be even better I think.

The reason it still used those is historical I guess, since it was together with streams, it had access there, and then we just "moved" the classes to new repo while it's access privileges to this class remained...